airgap_instructions.adoc

```diff
@@ -0,0 +1,206 @@
+CaaSP is sourced from three types of repositories:
+
+. RPM packages - MicroOS
+.. Software updates on the operating system
+.. Provided by SUSE package repositories
+. Container images - Docker, CRI-O
+.. Software updates for the deployments and the CaaSP infrastructure layer
+.. Provided by Docker on dockerhub
+.. Provided by Google on k8s.io
```

Comment:
We don't pull anything neither from the docker hub nor from k8s.io. You might mention `registry.suse.com`.

Reply:
I'll remove that file that is an outdated notes file